### ISSUE TITLE
Change map display for waypoint view

### ISSUE THREAD

**Issue by Oskar1983** [cgeo/cgeo#6970]
Hello

is it possible to change the map display when viewing a waypoint of a multicache that way that also other caches are shown? At the moment you only see the waypoint, but it would be helpful to see other caches (and maybe the live map?) as well when being nearby.

Thank you!

**Comment by Lineflyer** [comment]
You could use the setting Menu/Settings/Map/"Show Waypoints" to enable (or raise the limit) so that waypoints of caches are shown on the normal live map.
I think it should deliver what you expect.

**Comment by Oskar1983** [comment]
No, that's not the point. I just had a look. There are three kinds of display and of of them are slightly different: 
1. Live map: having all caches and waypoint shown als long as the limit is not reached.
2. Map of cache: showing waypoints of selected cache and cache itself. If "enable live" ist pressed, other caches and waypoints are shown as well. I assume without respecting the limt - is that intended to work this way? That's somehow weird, especially displaying stored caches only when "enable live" is pressed.
3. Map of waypoint: showing only the selected waypoint and having a brouter route to it. Nothing else is shown and nothing can be switched or configured. 

That's somehow inconsistent and I think it would be nice to have at least an option to have all map displays behave the same way.

**Comment by Lineflyer** [comment]
> I assume without respecting the limt - is that intended to work this way? That's somehow weird, especially displaying stored caches only when "enable live" is pressed.

The limit should be respected in this case. 
Why is it weird to display stored caches when you enable live. Those are live as well, just stored on your device. It will show those and also all non stored caches.

> Map of waypoint: showing only the selected waypoint and having a brouter route to it. Nothing else is shown and nothing can be switched or configured.

I  still don't understand why you dont use one of the other map types you described in your post to get what you need?

**Comment by Oskar1983** [comment]
> The limit should be respected in this case.

It isn't - see e.g. gc4ve71: 
gc4eyw7 is nearby. That one has a lot of waypoints. When zooming out in live map those are displayed only when below the limit - in cache map they are displayed all the time without regarding the limit.

> Why is it weird to display stored caches when you enable live. Those are live as well, just stored on your device. It will show those and also all non stored caches.

Using "enable live" is (not only) for me the association of an online "live" display. The stored caches should also be shown without "enabling live". Which is the case for live map but not for the cache map. That's somehow inconsistent.

>I still don't understand why you dont use one of the other map types you described in your post to get what you need?

Because you are just not able to change the map type when you are going to a waypoint of a cache.

**Comment by rsudev** [comment]
While the topic of consistence across all map types is still to be discussed, I think the use-case you have in mind can be much better served with the new map.
Here you can open the map from a cache, activate live and than select any waypoint or cache as new navigation target in the pop-up. You can either activate it as default or select it in the navigation menu of a cache to give it a try.
